ibis/expr/datatypes.py

```diff
@@ -1040,9 +1100,9 @@ def type(self):
             raise SyntaxError('Type cannot be parsed: {}'.format(self.text))
 
 
-def array_type(t):
+def column_type(t):
```

Comment:
I think We can remove it, seems unused: https://github.com/ibis-project/ibis/search?utf8=%E2%9C%93&q=array_type&type=

Reply:
Agreed.

Reply:
Done